src/main/java/com/rapid7/client/dcerpc/mssamr/messages/SamrOpenLocalGroupRpcRequest.java

```diff
@@ -0,0 +1,49 @@
+/***************************************************************************
```

Comment:
Wrong copyright header.

I made the same mistake.

Reply:
yep, I will  update it